compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -242,6 +251,55 @@ void buildHALTransformPassPipeline(OpPassManager &passManager,
   if (compileTo == PipelinePhase::ExecutableSources)
     return;
 
+  //----------------------------------------------------------------------------
+  // Executable translation
+  //----------------------------------------------------------------------------
+
+  if (compileFrom < PipelinePhase::ExecutableConfigurations) {
+    // Select a strategy for each hal.executable.variant and generate the IR to
```

Comment:
".. a translation strategy .." to be clear and relevant to hal. I guess we should also look at other places where we only mention "strategy" by itself and do the same when suitable. (I've commented at a few other places but not exhaustive for sure.)